docs/django-apps.md

```diff
@@ -80,6 +80,21 @@ if DEBUG:
 ```
 
 
+### urls.py
+
+Include the Caluma urls:
+
+```python
+from django.conf.urls import include, url
+
+urlpatterns = [
+    # ...
+    url(r"", include("caluma.caluma_core.urls")),
```

Comment:
Add a suggestion for  `/graphql` as URL prefix for the include here

Reply:
Done.